tests/test_tools.py

```diff
@@ -189,3 +189,65 @@ def test_get_package_info(monkeypatch, inputs, expected):
     )
     actual_metadata = get_package_info(inputs[0], metadata=inputs[1])
     assert actual_metadata == expected
+
+
+@pytest.mark.parametrize(
+    "inputs, expected_mu",
+    [
+        # Test whether the function returns the correct mu
```

Comment:
reorganized test comments

Reply:
I didn't add test for implementation error - shall I add one?

Reply:
no, we don't need a test for that.